utils/R/convert.input.R

```diff
@@ -8,7 +8,7 @@
 
 convert.input <- function(input.id, outfolder, formatname, mimetype, site.id, start_date, 
                           end_date, pkg, fcn, con = con, host, browndog, write = TRUE, 
-                          format.vars, overwrite = FALSE, exact.dates = FALSE, ...) {
+                          format.vars, overwrite = FALSE, exact.dates = FALSE, allow.conflicting.dates = TRUE, ...) {
```

Comment:
I'm missing what allow.conflicting.dates actually means. By name, it seems to just be the opposite of exact.dates, but neither is documented